ui/app/components/mount-backend/type-form.hbs

```diff
@@ -4,39 +4,30 @@
 ~}}
 
 {{#each (array "generic" "cloud" "infra") as |category|}}
-  <h3 class="title box-radio-header">
```

Comment:
I think we can remove `box-radio-header` in the `box-radio.scss` file since `type-form` is the only place that used this class.

Reply:
I'll be deleting all of the box radio styling + component at the end (for diff purposes waited so the review was easier with fewer files 😄 )